src/reducers/monitors.js

```diff
@@ -0,0 +1,25 @@
+const UPDATE_MONITORS = 'scratch-gui/monitors/UPDATE_MONITORS';
+
+const initialState = [];
+
+const reducer = function (state, action) {
+    if (typeof state === 'undefined') state = initialState;
+    switch (action.type) {
+    case UPDATE_MONITORS:
+        return action.monitors;
```

Comment:
I think the best practice is to always make a copy like `return [... action.monitors]`.  I was going to say in practice this won't be too important since we'll be making a copy in the VM. But we currently don't do that with targets, so I think we probably should copy here.